tests/simpleKVBC/TesterCRE/main.cpp

```diff
@@ -23,8 +23,20 @@
 #include "client/reconfiguration/config.hpp"
 #include "client/reconfiguration/poll_based_state_client.hpp"
 #include "client/reconfiguration/client_reconfiguration_engine.hpp"
+#include "crypto_utils.hpp"
+#include "secrets_manager_plain.h"
 #include <variant>
 
+#if __has_include(<filesystem>)
```

Comment:
filesystem is a part of std++17
no need in ifdef I believe

Reply:
fixed